distributed/diagnostics/plugin.py

```diff
@@ -287,7 +293,10 @@ async def setup(self, worker):
             await Semaphore(max_leases=1, name=socket.gethostname(), register=True)
         ):
             if not await self._is_installed(worker):
-                logger.info("Pip installing the following packages: %s", self.packages)
+                logger.info(
+                    f"{self.installer} installing the following packages: %s",
+                    self.packages,
```

Comment:
```suggestion
                    f"%s installing the following packages: %s",
                    self.installer, self.packages,
```
nit: don't mix-n-match %-strings and f-strings